src/java.base/share/native/libfallbackLinker/fallbackLinker.c

```diff
@@ -165,7 +165,7 @@ Java_jdk_internal_foreign_abi_fallback_LibFallback_doDowncall(JNIEnv* env, jclas
 static void do_upcall(ffi_cif* cif, void* ret, void** args, void* user_data) {
   // attach thread
   JNIEnv* env;
-  jint result = (*VM)->AttachCurrentThreadAsDaemon(VM, (void**) &env, NULL);
+  (*VM)->AttachCurrentThreadAsDaemon(VM, (void**) &env, NULL);
```

Comment:
I think the variable should be used! This code should be checking for a failed attach!

Reply:
Agree with David here. Generally I comment out unused code errors rather than outright remove them for this very reason. Also fallbackLinker doesn't seem to be a C\+\+ file?

Reply:
How about the following?

```
  jint result = (*VM)->AttachCurrentThreadAsDaemon(VM, (void**) &env, NULL);
  if (result != JNI_OK) {
    fprintf(stderr, "do_upcall trying to attach thread returned %d", result);
    return;
  }
```

Reply:
We can not return to native code here in case of an error. We should terminate the process instead.

The same should really also be done in case the call to `CallStaticVoidMethod` below throws an exception. This was a leftover from the initial implementation of this code that apparently fell through the cracks.

Ideally we should also add a test for that so... I suggest just removing the unused variable in this PR for now, to get this to compile, and I then I can file a followup which addresses the lack of error handling.

Reply:
Thanks @JornVernee. The current change is just removing the unused variable. Could you approve it?

Reply:
Ok, I've filed: https://bugs.openjdk.org/browse/JDK-8346575 for addressing the error handling.